frontend/server/README

```diff
@@ -0,0 +1,3 @@
+In this folder we have files representing the sample output of our backend.
+Those files are used to mock backend.
+The name of files represent the url's suffix.
```

Comment:
s/url's suffix/backend method/

Reply:
Generally these LGTM as mocks, but the actual data format will almost certainly change when the metabackend comes to life :).